core/src/main/resources/defaults/blueprints/7.2.16/cdp-sdx.bp

```diff
@@ -275,7 +275,13 @@
           {
             "base": true,
             "refName": "hdfs-GATEWAY-BASE",
-            "roleType": "GATEWAY"
+            "roleType": "GATEWAY",
+            "configs": [
+              {
+                "name": "hdfs_client_config_safety_valve",
+                "value": "<property><name>dfs.client.block.write.replace-datanode-on-failure.policy</name><value>NEVER</value></property><property><name>dfs.client.block.write.replace-datanode-on-failure.enable</name><value>FALSE</value></property>"
```

Comment:
I don't know if Hadoop is sensitive to the case of a boolean value, so I would recommend setting it to "false" (instead of FALSE) to be on the safer side.